changelogs/8.18.asciidoc

```diff
@@ -1,7 +1,32 @@
 [[apm-release-notes-8.18]]
 == APM version 8.18
+
+* <<apm-release-notes-8.18.1>>
 * <<apm-release-notes-8.18.0>>
 
+[float]
+[[apm-release-notes-8.18.1]]
+=== APM version 8.18.1
+
+https://github.com/elastic/apm-server/compare/v8.18.0\...v8.18.1[View commits]
+
+[float]
+==== Bug fixes
+
+- Tail-based sampling: Fix an issue with corrupted subscriber position files by handling file read error {pull}16736[16736]
```

Comment:
```suggestion
- Tail-based sampling: ignore subscriber position read error and proceed as if file does not exist to avoid crash looping {pull}16736[16736]
```
nit on changelog line